packages/stashjs-examples/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist",
+    "rootDir": ".",
+    "tsBuildInfoFile": "./dist/tsconfig.tsbuildinfo",
+
+    // Relaxed rules to allow package to build. FIXME: remove these and fix in code
```

Comment:
When do you think we should fix these up? As part of the blanket card for similar fixes for the other packages in this repo?

Reply:
Good question - might as well do it now.